com.usebottles.bottles.yml

```diff
@@ -0,0 +1,69 @@
+---
+app-id: com.usebottles.bottles
+runtime: org.gnome.Sdk
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+add-extensions:
+  org.gnome.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '3.38'
+command: bottles
+finish-args:
+- --allow=multiarch
+- --share=network
+- --device=dri
+- --share=ipc
+- --socket=fallback-x11
+- --socket=wayland
```

Comment:
Do Wine applications work natively on Wayland these days? If not, I'd stick to `x11`.

Reply:
No wine doesn't work with wayland at this time, my fault